doc/base/classes.xml

```diff
@@ -58,7 +76,11 @@
 			<argument index="0" name="s" type="float">
 			</argument>
 			<description>
-				Returns the principal value of the arc sine of s, expressed in radians. In trigonometrics, arc sine is the inverse operation of sine.
+				Returns the arc sine of 's' in radians. The angle in radians whose sine is 's'.
```

Comment:
"whose" is more for people. I'd use simpler language: `Use it with a cosine to get an angle`, or `Useful to get an angle from a cosine value`, something like this.